Rocket.Chat/Views/Chat/New Chat/Cells/AudioMessageCell.swift

```diff
@@ -2,15 +2,16 @@
 //  AudioMessageCell.swift
 //  Rocket.Chat
 //
-//  Created by Rafael Streit on 28/09/18.
+//  Created by Filipe Alvarenga on 15/10/18.
```

Comment:
:trollface:

Reply:
Hahah, this is because I refactored the former `AudioMessageCell` to `AudioCell` then created a new file called `AudioMessageCell` but the diff couldn't track it right